contracts/script/GenerateAlloc.s.sol

```diff
@@ -159,37 +159,33 @@ contract GenerateAlloc is Script {
         vm.resetNonce(tmp);
 
         InitializableHelper.disableInitializers(impl);
-        IPTokenStaking(Predeploys.Staking).initialize(protocolAdmin, 1 ether, 1 ether, 1 ether, 7 days);
+        IIPTokenStaking.InitializerArgs memory args = IIPTokenStaking.InitializerArgs({
+            owner: protocolAdmin,
+            minStakeAmount: 1 ether,
+            minUnstakeAmount: 1 ether,
+            minCommissionRate: 5_00, // 5% in basis points
+            shortStakingPeriod: 1 days, // TBD
+            mediumStakingPeriod: 2 days, // TBD
+            longStakingPeriod: 3 days, // TBD
+            unjailFee: 1 ether
+        });
+
+        // Testnet timing values
+        if (block.chainid != MAINNET_CHAIN_ID) {
+            args.minCommissionRate = 5_00 seconds;
+            args.shortStakingPeriod = 10 seconds;
+            args.mediumStakingPeriod = 15 seconds;
+            args.longStakingPeriod = 20 seconds;
```

Comment:
where are these periods be used?

Reply:
They are passed to the initializer, and used to resolve staking duration in `stake`